symfony/framework-bundle/5.1/config/services.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use App\Kernel;
+use Symfony\Component\DependencyInjection\Loader\Configurator as di;
+
+// This file is the entry point to configure your own services.
+// Files in the packages/ subdirectory configure your dependencies.
+
+return function (di\ContainerConfigurator $container, Kernel $kernel): void {
+
+    // Parameters are configuration that don't need to change depending on the machine where the app is deployed.
+    // see https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+    $container->parameters()
+    //    ->set(...)
+    ;
+
+    $services = $container->services()
+        ->defaults()
+            ->autowire()
+            ->autoconfigure()
+    ;
+
+    // Makes classes in src/ available to be used as services
+    $services
+        ->load('App\\', '../src/*')
+        ->exclude('../src/{DependencyInjection,Entity,Migrations,Tests,Kernel.php}');
```

Comment:
What is the best:

:+1::
```php
        ->exclude('../src/{DependencyInjection,Entity,Migrations,Tests,Kernel.php}');
```

:smile::
```php
        ->exclude(['../src/DependencyInjection', '../src/Entity', ...,]');
```

:tada::
```php
        ->exclude('../src/DependencyInjection')
        ->exclude('../src/Entity')
        ->exclude('../src/Migrations')
        ->exclude('../src/Tests')
        ->exclude('../src/Kernel.php')
```
---

I really like the last one (:tada:) because:

* it demonstrates it's possible ([ref](https://github.com/symfony/symfony/issues/30865))
* it scales really well
* the diff is not messed up when editing one directory
* it's simpler to find / edit something

Reply:
We should pick the alternative that actually works, I don't remember which one does :)

Reply:
Multiple-excludes don't work: the next one replaces the previous one.
But an array work. updated.